.eslintrc.js

```diff
@@ -1,17 +1,61 @@
 const { overrides } = require('@netlify/eslint-config-node')
 
 module.exports = {
-  extends: '@netlify/eslint-config-node',
+  extends: ['@netlify/eslint-config-node', 'plugin:node/recommended'],
   rules: {
+    'node/handle-callback-err': 2,
+    'node/no-new-require': 2,
+    'node/exports-style': 2,
+    'node/file-extension-in-import': 2,
+    'node/no-mixed-requires': 2,
+    // Browser globals should not use `require()`. Non-browser globals should
+    'node/prefer-global/console': 2,
+    'node/prefer-global/text-decoder': 2,
+    'node/prefer-global/text-encoder': 2,
+    // This does not work well in a monorepo
+    'node/shebang': 0,
+
+    // TODO: enable this
```

Comment:
I will add separate PRs for those rules.
Except for `no-process-exit`, since the changes might be too big.